EMpy/modesolvers/FD.py

```diff
@@ -981,13 +986,14 @@ def solve(self, neigs=4, tol=0, guess=None):
         else:
             shift = None
 
+        # Here is where the actual mode-solving takes place! 
         [eigvals, eigvecs] = eigen.eigs(A,
-                                        k=neigs,
```

Comment:
Again, this is not PEP8 compliant. In this case, I think it's more elegant to leave no spaces around the "equal" signs.